src/index.html

```diff
@@ -26,54 +26,42 @@
+              Dart helps you craft beautiful, high-quality experiences
+              across all screens, with:
+
+              <ul>
+                <li> A <span data-toggle="popover" data-content="Dart started out being optimized for web apps, and is evolving to provide great support for mobile apps. Dart also runs on the command line and server-side.">client-optimized</span> language </li>
+                <li> Rich, powerful <span data-toggle="popover" data-content="For mobile apps, we recommend <a href='https://flutter.io'>Flutter</a>. For desktop apps, <a href='https://webdev.dartlang.org/angular'>AngularDart</a>.">frameworks</span> </li>
+                <li> Delightful, flexible <span data-toggle="popover" data-content="These range from <a href='/tools'>general-purpose tools</a> to framework-specific ones like the <a href='https://flutter.io/getting-started'><code>flutter</code> tool</a>.">tooling</span>
+                </li>
+              </ul>
+              <p><a href="/guides/get-started" class="frontpage-slogan__get-started btn btn-default btn-lg">Get Started</a> </p>
             </h3>
-            <p><a href="/guides/get-started" class="frontpage-slogan__get-started btn btn-default btn-lg">Get Started</a> <a href="/install" class="btn btn-lg">Install Dart</a></p>
+            <p> <em>Click <span data-toggle="popover" data-content="Try clicking dotted underlines in the text above and in the code sample.">underlined text</span> to learn more.</em> </p>
         </div>
       </div>
     </div>
   </div>
 </div>
 </section>
 
-<section class="chip-grid-section">
+<section class="card-grid-section">
   <div class="container">
     <div class="row">
       <div class="col-sm-12">
         <div class="content">
-          <div class="chip-grid">
-            <a href="http://news.dartlang.org/" class="chip no-automatic-external">
-              <h3>News</h3>
-              <p>Follow the latest.</p>
-            </a>
-            <a href="https://api.dartlang.org/" class="chip no-automatic-external">
-              <h3>API</h3>
-              <p>Browse core libraries.</p>
-            </a>
-            <a href="https://pub.dartlang.org/" class="chip no-automatic-external">
-              <h3>Pub</h3>
-              <p>Find packages.</p>
-            </a>
-            <a href="{{site.webdev}}" class="chip no-automatic-external">
-              <h3>Dart webdev</h3>
-              <p>Build browser apps.</p>
+          <div class="card-grid">
+            <a href="/guides/language/language-tour" class="card no-automatic-external">
+              <h4>Language tour</h4>
+              <p>A walk through Dart's major language features.</p>
             </a>
-            <a href="https://flutter.io/" class="chip no-automatic-external">
-              <h3>Flutter</h3>
-              <p>Build mobile apps.</p>
+            <a href="/guides/libraries/library-tour" class="card no-automatic-external">
```

Comment:
I'd prefer having the "Platform" (https://www.dartlang.org/guides/platforms) as one of the three boxes instead of the library tour. Ideally as the last one after Pub

Reply:
I'm OK with Platform as one of the 3, but I'd like the page to be more attractive. I'll file a bug for that.